phalcon/config.zep

```diff
@@ -223,6 +270,29 @@ class Config implements \ArrayAccess, \Countable
 	}
 
 	/**
+	 * Sets the default path delimiter
+	 */
+	public static function setPathDelimiter(string! delimiter = null) -> void
```

Comment:
It would be impossible/erroneous to use a config key named 'pathDelimiter' or '_delimiter' or what ever I would name the var to hold the path delimiter. `Phalcon\Config` implements ArrayAccess...

I could use a global constant PHALCON_CONFIG_PATH_DELIMITER to test for a changed default delimiter!?